keyboards/keycapsss/plaid_pad/keymaps/default_flipped/keymap.c

```diff
@@ -0,0 +1,44 @@
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+/*
+ * ,-----------------------,
+ * |  7  |  8  |  9  |  /  |
+ * |-----+-----+-----+-----|
+ * |  4  |  5  |  6  |  *  |
+ * |-----+-----+-----+-----|
+ * |  1  |  2  |  3  |  -  |
+ * |-----+-----+-----+-----|
+ * |  0  |  .  |  =  |  +  |
+ * `-----------------------'
+ */
+LAYOUT_ortho_4x4(
+		KC_P7, KC_P8,   KC_P9,      KC_PSLS,
+		KC_P4, KC_P5,   KC_P6,      KC_PAST,
+		KC_P1, KC_P2,   KC_P3,      KC_PMNS,
+		KC_P0, KC_PDOT, KC_PEQL,    KC_PPLS
+)
+
+};
+
+// Set led state during powerup
+void keyboard_post_init_user(void) {
+    writePinHigh(LED_RED);
+}
+
+void encoder_update_user(uint8_t index, bool clockwise) {
+  if (index == 0) {         // First encoder - top left
+    if (clockwise) {
+      tap_code(KC_RIGHT);
+    } else {
+      tap_code(KC_LEFT);
```

Comment:
Would change it to KC_RIGHT, because right now if i turn the knob ccw, the cursor moves to the right.
Don't forget to change it also in the flipped keymap.